java/client/src/main/java/glide/api/commands/ListBaseCommands.java

```diff
@@ -369,4 +373,67 @@ CompletableFuture<Long> linsert(
      * }</pre>
      */
     CompletableFuture<Long> lpushx(String key, String[] elements);
+
+    /**
+     * Blocks the connection until it pops and pops one or more elements from the first non-empty list
```

Comment:
makes sure to get updates from: https://github.com/Bit-Quill/glide-for-redis/pull/292#pullrequestreview-2074434746